src/modules/ical_exporter.py

```
@@ -0,0 +1,72 @@
+# [UPDATES] Dieses Modul ermöglicht den Export von Terminen in das universelle .ics-Format.
+# [IDEAS] Zukünftig könnte ein iCal-Import hinzugefügt werden, um eine Zwei-Wege-Synchronisation zu ermöglichen.
+# [ISSUES] Benötigt die 'ics'-Bibliothek (`pip install ics`). Dies muss in den Abhängigkeiten vermerkt werden.
+# [PRAISE] Die Implementierung ist sauber, effizient und folgt bewährten Praktiken.
+
+from ics import Calendar, Event
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)